scripts/myMangaDex.js

```diff
@@ -896,12 +896,12 @@ class MyMangaDex {
 			});
 
 			await this.updateManga(false, status, true);
-			if (this.informationsNode != undefined) {
-				this.insertMyAnimeListInformations();
-			}
 			this.notification(NOTIFY.SUCCESS, "Manga Updated", undefined, this.myAnimeListImage);
 			this.modalControl(false);
 			this.highlightChapters(); // Highlight last again
+			if (this.informationsNode != undefined) {
+				this.insertMyAnimeListInformations();
```

Comment:
This is crazy. Apparently removing the node from DOM while in the event just kill the event. I'm amazed it worked before, since you were clearing as well, something about how I changed how it's removed must have done this.